doc/user_doc/vic_installation/plugin_h5_vc_web.md

```diff
@@ -0,0 +1,32 @@
+# Install the HTML5 Plug-In on vCenter Server for Windows by Using a Web Server #
+
+If your vCenter Server instance runs on Windows, you can use a Web server to host the HTML5 vSphere Client plug-in for vSphere Integrated Containers Engine.
+
+**Prerequisites**
+
+- You are running a Web server that your vCenter Server instance can access.
+- Use a Windows system to run the script to install the plug-in on a vCenter Server that runs on Windows. If you used a Linux or Mac OS system to deploy the VCH, download and unpack the vSphere Integrated Containers Engine package on a Windows system. For example, download the package to the system on which vCenter Server is running.
+
+**Procedure**
+
+1. On the Windows system on which you have downloaded and unpacked vSphere Integrated Containers Engine, navigate to the folder that contains the `vic-machine` utility and open the `ui` folder.
+2. Upload the plug-in bundle to your Web server.
+    <pre><i>unpack_dir</i>\vic\ui\plugin-packages\com.vmware.vic-<i>version</i>.zip</pre>
+3. On the system on which the Web server is running, make sure that port 3443 is open for both inbound and outbound connections.
```

Comment:
Updated to "On the system on which the Web server is running, if you do not use port 443 for secure HTTP traffic, make sure that the appropriate port is open for both inbound and outbound connections."